Kudu.Core/Environment.cs

```diff
@@ -270,5 +273,40 @@ public static bool IsAzureEnvironment()
         {
             return !String.IsNullOrEmpty(System.Environment.GetEnvironmentVariable("WEBSITE_INSTANCE_ID"));
         }
+
+        public static string GetFreeSpaceHtml(string path)
+        {
+            try
+            {
+                ulong freeBytes;
+                ulong totalBytes;
+                GetDiskFreeSpace(path, out freeBytes, out totalBytes);
+
+                var usage = Math.Round(((totalBytes - freeBytes) * 100.0) / totalBytes);
+                var color = usage > 97 ? "red" : (usage > 90 ? "orange" : "green");
+                return String.Format("<span style='color:{0}'>{1:#,##0} MB total; {2:#,##0} MB free</span>", color, totalBytes / (1024 * 1024), freeBytes / (1024 * 1024));
```

Comment:
for safety, would be good to change String.Format to String.Format(***CultureInfo.InvariantCulture***...